addons/website/models/ir_ui_view.py

```diff
@@ -25,8 +25,19 @@ class View(models.Model):
     first_page_id = fields.Many2one('website.page', string='Website Page', help='First page linked to this view', compute='_compute_first_page_id')
     track = fields.Boolean(string='Track', default=False, help="Allow to specify for one page of the website to be trackable or not")
     visibility = fields.Selection([('', 'All'), ('connected', 'Signed In'), ('restricted_group', 'Restricted Group'), ('password', 'With Password')], default='')
-    visibility_group = fields.Many2one('res.groups')
-    visibility_password = fields.Char(groups='website.group_website_publisher')
+    visibility_group = fields.Many2one('res.groups', copy=False)
+    visibility_password = fields.Char(groups='base.group_system', copy=False)
```

Comment:
I don't know if the goal is for system admins to see this or for it to be as invisible as possible, but if the latter an option @odony gave me which is slightly more secure (without being as good as an off-model SQL field) is
```suggestion
    visibility_password = fields.Char(groups='...', copy=False)
```
as that's not a real group it pretty much requires a sudo() access in order to bypass groups / ACL entirely.

Reply:
@mart-e can you explain more ? . or ... is not valid group, so what you suggest ?

Reply:
It's a bit hacky but it works
```python
>>> self.user_has_groups("no.one")
False
```
Maybe this is the sign we need a flag "sudo-only" on a field though...

Reply:
> It's a bit hacky but it works

Until someone creates a group no.one.